web/src/portico/landing-page.ts

```diff
@@ -8,100 +8,117 @@ import iphone_image from "../../images/app-screenshots/zulip-iphone-rough.png";
-            app_type: "desktop",
         },
         android: {
-            image: android_image,
             alt: "Android",
+            app_type: "mobile",
             description: "Zulip's native Android app makes it easy to keep up while on the go.",
-            show_instructions: false,
-            play_store_link: "https://play.google.com/store/apps/details?id=com.zulipmobile",
             download_link: "https://github.com/zulip/zulip-mobile/releases/latest",
-            app_type: "mobile",
+            image: android_image,
+            play_store_link: "https://play.google.com/store/apps/details?id=com.zulipmobile",
+            show_instructions: false,
         },
         ios: {
-            image: iphone_image,
             alt: "iOS",
-            description: "Zulip's native iOS app makes it easy to keep up while on the go.",
-            show_instructions: false,
             app_store_link: "https://itunes.apple.com/us/app/zulip/id1203036395",
             app_type: "mobile",
+            description: "Zulip's native iOS app makes it easy to keep up while on the go.",
+            image: iphone_image,
+            show_instructions: false,
         },
         linux: {
-            image: ubuntu_image,
             alt: "Linux",
+            app_type: "desktop",
             description:
                 "Zulip on the Linux desktop is even better than Zulip on the web, with a cleaner look, tray integration, native notifications, and support for multiple Zulip accounts.",
             download_link: "/apps/download/linux",
-            show_instructions: true,
+            image: ubuntu_image,
             install_guide: "/help/desktop-app-install-guide",
-            app_type: "desktop",
+            show_instructions: true,
         },
     };
 
-    let version;
+    let version: UserOS;
 
-    function get_version_from_path() {
-        let result;
+    function get_version_from_path(): UserOS {
+        let result: UserOS | undefined;
         const parts = path_parts();
 
-        for (const version of Object.keys(info)) {
+        for (const version of Object.keys(info) as UserOS[]) {
             if (parts.includes(version)) {
                 result = version;
             }
         }
 
-        result = result || detect_user_os();
-        return result;
+        return result ?? detect_user_os();
     }
 
-    const update_page = function () {
+    const update_page = function (): void {
         const $download_instructions = $(".download-instructions");
         const $third_party_apps = $("#third-party-apps");
         const $download_android_apk = $("#download-android-apk");
         const $download_from_google_play_store = $(".download-from-google-play-store");
         const $download_from_apple_app_store = $(".download-from-apple-app-store");
         const $download_mac_arm64 = $("#download-mac-arm64");
         const $desktop_download_link = $(".desktop-download-link");
+
```

Comment:
Please pay attention to newlines -- I don't think there's a reason to add one between these two items.